File: src/pycropml/transpiler/antlr_py/tests/examples/SimplaceComponent/SoilTemp/src/cs/SoilTemperature/SoilTemperatureState.cs
using System;
using System.Collections.Generic;
public class SoilTemperatureState 
{
    private double _SoilSurfaceTemperature;
    private double _SnowWaterContent;
    
    public SoilTemperatureState() { }
    
    
    public SoilTemperatureState(SoilTemperatureState toCopy, bool copyAll) // copy constructor 
    {
    if (copyAll)
    {
    
    _SoilSurfaceTemperature = toCopy._SoilSurfaceTemperature;
    _SnowWaterContent = toCopy._SnowWaterContent;
    }
    }
    public double SoilSurfaceTemperature
        {
            get { return this._SoilSurfaceTemperature; }
            set { this._SoilSurfaceTemperature= value; } 
        }
    public double SnowWaterContent
        {
            get { return this._SnowWaterContent; }
            set { this._SnowWaterContent= value; } 
        }
}

File: src/pycropml/transpiler/antlr_py/tests/examples/SimplaceComponent/SoilTemp/src/java/SoilTemperature/SoiltemperatureState.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;
public class SoiltemperatureState
{
    private double SoilSurfaceTemperature;
    private double SnowWaterContent;
    
    public SoiltemperatureState() { }
    
    public SoiltemperatureState(SoiltemperatureState toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.SoilSurfaceTemperature = toCopy.getSoilSurfaceTemperature();
            this.SnowWaterContent = toCopy.getSnowWaterContent();
        }
    }
    public double getSoilSurfaceTemperature()
    { return SoilSurfaceTemperature; }

    public void setSoilSurfaceTemperature(double _SoilSurfaceTemperature)
    { this.SoilSurfaceTemperature= _SoilSurfaceTemperature; } 
    
    public double getSnowWaterContent()
    { return SnowWaterContent; }

    public void setSnowWaterContent(double _SnowWaterContent)
    { this.SnowWaterContent= _SnowWaterContent; } 
    
}